integration_tests/src/main/python/iceberg/iceberg_test.py

```diff
@@ -64,16 +67,19 @@ def setup_iceberg_table(spark):
                     reason="AQE+DPP not supported until Spark 3.2.0+ and AQE+DPP not supported on Databricks")
 @pytest.mark.parametrize('reader_type', rapids_reader_types)
 def test_iceberg_aqe_dpp(spark_tmp_table_factory, reader_type):
+    ns = setup_namespace(spark_tmp_table_factory)
     table = spark_tmp_table_factory.get()
+    full_table = f"{ns}.{table}"
```

Comment:
If it's not s3 table, then `full_table` is `spark_tmp_table_factory.get()`.`spark_tmp_table_factory.get()`, is it expected?
Maybe add a function:
```python
def get_full_qualified_table_name(spark_tmp_table_factory):
  tab = spark_tmp_table_factory.get()
  if s3 table:
     return s3_table_ns.tab
  else:
     return tab
```